apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/ColumnQualifier.cs

```diff
@@ -130,6 +131,15 @@ public override string ToString()
         /// 
         /// </summary>
         /// <param name="qualifier"></param>
+        public static implicit operator string(ColumnQualifier qualifier)
```

Comment:
Not sure about the conversions this way being implicit. Let's discuss it when we get further along.
(Particularly for this one, which will presumably throw if the qualifier key isn't actually UTF-8-encoded text.)

Reply:
Oops. More copy/paste errors. These should have been explicit. Fixed the `RowKey` ones as well.